adafruit_character_lcd/character_lcd.py

```diff
@@ -198,6 +203,20 @@ def clear(self):
         self._write8(_LCD_CLEARDISPLAY)
         time.sleep(0.003)
 
+    @property
+    def column_align(self):
+        """If True, message text after '\n' starts directly below start of first
+        character in message. If False, text after '\n' starts at column zero.
```

Comment:
Been looking into this since you submitted changes. It looks like it's failing in travis because the \n's need to be escaped. Instead of '\n' use '\\n'

Reply:
Done